gf-webapp/src/logic/rooms/roomsPage/roomBlock/RoomBlock.module.scss

```diff
@@ -1,195 +0,0 @@
-@import "../../../../Index.scss";
```

Comment:
All these styles aren't used? Or why this file was deleted?

Reply:
Yes, this file doesn't import anywhere. Instead of it, there is file gf-webapp/src/logic/rooms/roomsPage/roomBlock/roomItem/RoomItem.module.scss with completely the same styles